server/routes/main.js

```diff
@@ -29,11 +29,20 @@ router.get(
 //The req.user, null, or a string "User is not in 100Devs"
 //Considering a string is truthy this .get distinguishes between the string and req.user
 router.get("/getDisplayName", (req, res) => {
+  console.log(req.user)
```

Comment:
this as well!

Reply:
no.

Reply:
Ok cool. I can get it merged once the extraneous console.logs are removed.